README.md

```diff
@@ -0,0 +1,26 @@
+# SDL 2.0.9 w/ Bazel build
+
+This project is WIP.
+
+# macOS build:
+
+```
+bazel build //:SDL2_dylib
+```
+
+I managed to get a project that uses SDL2 to work with the SDL2.dylib built by bazel.
+
+# Linux build:
+```
+bazel build //:libSDL2.so
+```
+
+The build succeeds, but I didn't try anything more than that.
+
+# Windows build:
+
+```
+bazel build //:SDL2.dll
+```
+
+The build succeeds, but I didn't try anything more than that.
```

Comment:
Could we remove this file from the the pull request? I want to keep this repository as similar to the original SDL2 source code as possible.

If we were to add a README I think I would like it to include installation instructions, but even then I'm unsure.

Reply:
Moved!

Reply:
You're right, having a README here would be nice for that. Feel free to keep it as README.md. I may make some change to it after this merge, however. :)